packages/doenetml-worker/package.json

```diff
@@ -46,20 +46,13 @@
         }
     },
     "peerDependencies": {
-        "react": "^18.2.0",
-        "react-dom": "^18.2.0",
+        "react": "^18.3.1",
+        "react-dom": "^18.3.1",
         "styled-components": "^5.3.11"
     },
     "dependencies": {
-        "compromise": "^13.11.4",
+        "compromise": "^14.13.0",
```

Comment:
Please revert back to 13.11.4. I haven't yet taken the time to adjust the `<pluralize>` tag for changes to `compromise`.